index.js

```diff
@@ -1008,31 +1053,42 @@ Multi.prototype.send_command = function (command, args, index, cb) {
     });
 };
 
-Multi.prototype.exec = Multi.prototype.EXEC = function (callback) {
+Multi.prototype.exec_transaction = function (callback) {
     var self = this;
+    var len = this.queue.length;
+    var cb;
```

Comment:
I'm not sure I follow - this case doesn't get handled at the moment and the redis server gets an empty `MULTI` `EXEC`, when we could immediately callback.

I've had to deal with multiple issues where there are empty multi blocks, and being able to call `.exec` without checking that `m.queue.length > 1` would be very useful (ideally, users would never want/need to play around with the internal queue).